src/main/java/com/ifmo/epampractice/daoimpl/OrderDaoImpl.java

```diff
@@ -0,0 +1,128 @@
+package com.ifmo.epampractice.daoimpl;
+
+import com.ifmo.epampractice.dao.DBConnectorInterface;
+import com.ifmo.epampractice.dao.DBConnectorPostgres;
+import com.ifmo.epampractice.dao.OrderDao;
+import com.ifmo.epampractice.entity.OrderEntity;
+import com.ifmo.epampractice.enums.OrderStatus;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
+
+import java.math.BigDecimal;
+import java.sql.*;
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.List;
+
+public class OrderDaoImpl implements OrderDao {
+    private static final String GET_ALL_QUERY = "SELECT * FROM ORDERS";
+    private static final String GET_QUERY = "SELECT * FROM ORDERS WHERE ID = ?";
+    private static final String SAVE_QUERY = "INSERT INTO ORDERS (car_id, client_id, admin_id, " +
+            "status, rent_start_date, rent_end_date, discount) VALUES (?, ?, ?, ?::e_status_order, ?, ?, ?)";
+    private static final String UPDATE_QUERY = "UPDATE ORDERS SET (car_id, client_id, admin_id, status, " +
+            "rent_start_date, rent_end_date, discount) = (?, ?, ?, ?::e_status_order, ?, ?, ?) WHERE ID = ?";
+    private static final String DELETE_QUERY = "DELETE FROM ORDERS WHERE ID = ?";
+
+    private static final Logger log = LogManager.getLogger(OrderDaoImpl.class);
+    private DBConnectorInterface dbConnector = DBConnectorPostgres.getInstance();
+
+    @Override
+    public List<OrderEntity> getAll() {
+        List<OrderEntity> orders = new ArrayList<>();
+        try (Connection connection = dbConnector.getConnection();
+             Statement statement = connection.createStatement(ResultSet.TYPE_SCROLL_SENSITIVE,
+                     ResultSet.CONCUR_READ_ONLY);
+             ResultSet resultSet = statement.executeQuery(GET_ALL_QUERY)) {
+            while (resultSet.next()) {
+                OrderEntity currentInvoice = parseRow(resultSet);
+                orders.add(currentInvoice);
+            }
+            return orders;
+        } catch (SQLException e) {
+            log.error(e);
+            throw new RuntimeException(e);
```

Comment:
I think logger will be enough for us, because it is not a good idea to throw a generic exception